<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogReplaceConfirmation</class>
 <widget class="QDialog" name="DialogReplaceConfirmation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>93</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Replace confirmation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Found an occurence of your search term.
What do you want to do?</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonReplace">
       <property name="text">
        <string>&amp;Replace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonReplaceAll">
       <property name="text">
        <string>Replace &amp;all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonFindNext">
       <property name="text">
        <string>&amp;Find next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClose">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>findNext()</slot>
  <slot>replaceAll()</slot>
  <slot>replace()</slot>
  <slot>closeDialog()</slot>
 </slots>
</ui>
